github/pulls_reviewers_test.go

```diff
@@ -66,8 +66,8 @@ func TestListReviewers(t *testing.T) {
 		t.Errorf("PullRequests.ListReviewers returned error: %v", err)
 	}
 
-	want := &[]User{
-		{
+	want := []*User{
+		&User{
```

Comment:
It's not neccessary to include `&User{` here, just `{` as before will do.

In fact, it causes Travis to fail because we check that `gofmt -s` doesn't find any missed opportunities to simplify:

![image](https://cloud.githubusercontent.com/assets/1924134/23475507/cfb29486-fe85-11e6-8caa-688af9b606fb.png)

The rest of this change is perfect.